scripts/springfield-buildings-list.csv

```diff
@@ -1,641 +0,0 @@
-"﻿ROLL_NUMBER","RSN","PID","LAT_LONG","PROPERTY_ADDRESS","PROPERTY_POSTAL_CODE","GEOCODE_ADDRESS","YEAR_BUILT","TYPE","PRIMARY_USE","DESCRIPTION","APPROX_BUILDING_AREA","HVAC","FLOORS","PROPERTY_BUILDINGS","NOTES","ORDER","EUI","ENERGY_STAR_SCORE","GREENHOUSE_GAS_EMISSIONS"
```

Comment:
Wait why did this disappear in an unrelated commit? We ought to just scrub this from history.